tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -4995,6 +4996,22 @@ def test_linspace2(start, end):
         x = linspace(start, end, 5)
         return x[0], x[1], x[2], x[3], x[4]
 
+    @types('int', 'int', 'int[:]')
+    def test_linspace_type(start, end, result):
+        from numpy import linspace
+        x = linspace(start, end, 15, dtype='int')
+        for i in range(len(x)):
+            if x[i] != result[i]:
+                return 0
+        return 1
```

Comment:
This test does not check if the type is correctly assigned, only if the code compiles. Could you please return an element of `x` and check that it is an integer

Reply:
updated